inc/datahandlers/post.php

```diff
@@ -618,8 +628,6 @@ function verify_prefix()
 		}
 		else
 		{
-			$prefix_cache = build_prefixes($prefix);
```

Comment:
Ah missed that. There should be a change though, there was at least one useless variable in this code part. Will recheck on monday too.